addon/webextension/content-script.js

```diff
@@ -18,6 +22,8 @@ myPort.onMessage.addListener(function(m) {
   }
 });
 
+const SUPPORTURL = "https://support.mozilla.org/kb/lookingglass";
```

Comment:
This page says "test - 12817 This article doesn't have approved content yet."; so it's not ready yet, but I assume it's the right page?

Reply:
that is!